docs/ui/organizations.md

```diff
@@ -12,17 +12,86 @@ tags:
-- Service accounts to generate credentials for specific workloads
-- Expanded and customizable [Flow Run Retention Policy](/ui/flow-runs/#flow-run-retention-policy)
+- Organizational accounts and membership management.
+- [Role Based Access Controls (RBAC)](/ui/roles/) to configure user permissions at organization and workspace levels.
+- Service accounts to generate credentials for specific workloads.
+- Expanded and customizable [Flow Run Retention Policy](/ui/flow-runs/#flow-run-retention-policy).
 
-![Organization member page in Cloud 2.](/img/ui/org-members.png)
+[Contact sales](https://www.prefect.io/pricing) to learn more about annual plans giving you control over organizations and user roles.
 
-Control granular access to workspaces by setting default access for all organization members, inviting specific members to collaborate, or adding service account permissions.
+## Navigating organizations
 
-![Organization workspace sharing in Cloud 2.](/img/ui/org-workspace-sharing.png)
+If you have access to organizations in Prefect Cloud, you can see the organizations you're a member of by selecting the **Organizations** icon in the left navigation bar.
 
-[Contact sales](https://www.prefect.io/pricing) to learn more about annuals plans and organizations.
+![Select the Organizations icon in Prefect Cloud.](/img/ui/organizations.png)
+
+When you select an organization, you'll see an overview of the organization.
+
+**Workspaces** shows you a list of workspaces within the organziation to which you have access.
+
+## Organization members
+
+**Members** shows you a list of users who are members of the organization. If you have been given the organization Admin role, you can invite new members and set organization roles for users here.
+
+![Viewing the Organization member page in Prefect Cloud.](/img/ui/org-members.png)
+
+Control granular access to workspaces by setting default access for all organization members, inviting specific members to collaborate in an organization workspace, or adding service account permissions.
+
+### Inviting organization members
+
+To invite new members to an organization in Prefect Cloud, select the **+** icon. Provide an email address and a default organization role. You may add multiple email addresses.
+
+![Invite new members to an organization in Prefect Cloud.](/img/ui/org-invite-members.png)
+
+The user will receive an invite email with a confirmation link at the address provided. If the user does not already have a Prefect Cloud account, they must create an account before accessing the organization.
+
+The organization role set for the invite is only a default for the new member. An Admin may configure a different role for the user as needed.
+
+The maximum number of organization members varies. [Contact sales](https://www.prefect.io/pricing) to learn more about annual plans giving you control over organizations and user roles.
```

Comment:
```suggestion
The maximum number of organization members varies. [Contact sales](https://www.prefect.io/pricing) to learn more about annual plans giving you control over organizations and user roles.
```

Reply:
Frankly, I owe you a followup PR on this later in the week, org account terms will be all self serve and we should add some more detail there. They dont have to contact sales at all anymore.